src/fileformats/ocd_file_export.cpp

```diff
@@ -2140,9 +2197,12 @@ void OcdFileExport::exportCombinedSymbol(OcdFile<Format>& file, const CombinedSy
 			return;
 		case Symbol::Line:
 			{
-				auto copy = duplicate(static_cast<const LineSymbol&>(*parts[0]));
+				auto const* main_line = static_cast<const LineSymbol*>(parts[0]);
```

Comment:
AFAIU, this code will never kick in. A few lines above we decrease the number of parts from two to one and the test for `combined_symbol` type is still `Symbol::Combined`. Therefore the switch goes through the `::Combined` branch and the flow goes to `exportGenericCombinedSymbol()`.